xmodule/capa/tests/helpers.py

```diff
@@ -58,10 +58,9 @@ def construct_callback(self, dispatch='score_update'):
         return dispatch
 
 
-def test_capa_system(render_template=None):
+def get_capa_system(render_template=None):
```

Comment:
```suggestion
def mock_capa_system(render_template=None):
```
This (or perhaps `get_mock_capa_system`) would be more clear and also match `mock_capa_block`.

Reply:
Thanks @wgu-taylor-payne I changed to the suggested change.